cordova-plugin-appcenter-shared/src/ios/AppCenterShared.m

```diff
@@ -12,6 +12,11 @@ + (void) setAppSecret: (NSString *)secret
     [MSAppCenter configureWithAppSecret:secret];
 }
 
++ (void) setUserId: (NSString *)userId
+{
```

Comment:
Formatting of this file looks wrong for me - half of the curly brackets in the same line and rest of it in a new line.
Could you please reformat it?

Reply:
I think it's they way ObjC should be formatted - Method curly brackets at the new line and `if` clause - same line. You can check for all `.m` files it's like that.